#! /bin/sh /usr/share/dpatch/dpatch-run
## no_rsvg_gz.dpatch by Julien Cristau <julien.cristau@ens-lyon.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: librsvg doesn't have a rsvg-gz.h header anymore, and
## DP: rsvg_handle_new_gz isn't necessary (svgz files are detected
## DP: automatically). So don't build a binding for this function.

@DPATCH@
diff -urNad lablgtk2-2.6.0~/config.make.in lablgtk2-2.6.0/config.make.in
--- lablgtk2-2.6.0~/config.make.in	2005-06-02 02:20:18.000000000 +0200
+++ lablgtk2-2.6.0/config.make.in	2006-02-09 19:09:26.000000000 +0100
@@ -18,7 +18,7 @@
 USE_GL=@USE_GTKGL@
 USE_GLADE=@USE_GLADE@
 USE_RSVG=@USE_RSVG@
-HAVE_SVGZ=@HAVE_SVGZ@
+HAVE_SVGZ=
 USE_GNOMECANVAS=@USE_GNOMECANVAS@
 USE_GNOMEUI=@USE_GNOMEUI@
 USE_PANEL=@USE_PANEL@
